src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -308,22 +309,39 @@ private void handleInvalidFileError(
             null
         );
 
-        ErrorMapping
-            .getFor(cause.getClass())
-            .ifPresent(errorCode -> {
-                try {
-                    sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
-                } catch (Exception exc) {
-                    log.error(
-                        "Error sending notification to the queue."
-                            + "File name: " + zipFilename + " "
-                            + "Container: " + containerName,
-                        exc
-                    );
-                }
-            });
+        Optionals.ifPresentOrElse(
+            ErrorMapping.getFor(cause.getClass()),
+            (errorCode) -> trySendErrorMessageAndMoveFileToRejectedContainer(
+                zipFilename,
+                containerName,
+                cause, eventId,
+                leaseId,
+                errorCode
+            ),
+            () -> blobManager.tryMoveFileToRejectedContainer(zipFilename, containerName, leaseId)
```

Comment:
> I asked for this after some changes were made in this PR, resulting in the lack of failure in case of misconfiguration.
Sorry, I was wrong - the problem was there even before your changes. We should throw an exception here. The most important change in this PR is to not catch exceptions thrown when we fail to send message.

Reply:
Oh! I have thought you meant to raise another task for this